### ISSUE TITLE
The issue about acc

### ISSUE THREAD

**Issue by nrryu** [cszhangzhen/HGP-SL#18]
Hi! I like your work. So I downloaded the code and ran it 5 times on my desktop. And I found the performance for the ENZYMES dataset looks strange. The seed and the accuracy of the HGP are the followings

0. 0.433333333333333
1. 0.266666666666666
2. 0.266666666666666
3. 0.366666666666666
4. 0.283333333333333

I'm using Python 3.7.4, pytorch==1.3.0, torch-scatter==1.4.0, torch-sparse==0.4.3, torch-cluster==1.4.5 and torch-geometric==1.3.2 as you recommend in the code. What I changed are hyperparameters and the number of layers as you suggested. I must have missed something.. But I don't know what it is. Can you help me?
main.py
![image](https://user-images.githubusercontent.com/73467425/167264332-4f521cc5-ea34-4883-8bf7-b75e409dee54.png)
model.py
![image](https://user-images.githubusercontent.com/73467425/167264354-69fac391-8db7-4fe4-ae18-706c17ec76a3.png)

**Comment by cszhangzhen** [comment]
Hi,

Thanks for your interest in our paper. The ENZYMES dataset is very small. Try to use less linear layers and also refer [here ](https://github.com/cszhangzhen/HGP-SL/issues/5#issuecomment-687030934)for more information.

**Comment by nrryu** [comment]
Oh, I didn't know the ENZYMES dataset had a problem. I will try less linear layers. Thank you for your answer :)
